modules/enet/doc_classes/ENetConnection.xml

```diff
@@ -73,7 +75,8 @@
 			<param index="4" name="in_bandwidth" type="int" default="0" />
 			<param index="5" name="out_bandwidth" type="int" default="0" />
 			<description>
-				Create an ENetHost like [method create_host] which is also bound to the given [param bind_address] and [param bind_port].
+				Creates an ENetHost like [method create_host], but bound it to the given [param bind_address] and [param bind_port].
```

Comment:
slight reword, also not a pain point, I can revert if u want :)

Reply:
[We tend to avoid "but" when possible](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#ban-these-8-words).

Reply:
I gave it a second and third read _but_ couldn't find any problematic nuance on this specific one :monocle_face:.

`but` is used there to contrast `create_host` with `create_host_bound`, I'm open to any suggestions tho :) .

Reply:
I am personally not a big fan of "see another method first to know what this method does". Ideally, method descriptions should be self-explanatory. In every other PR of mine I replaced these kinds of descriptions with proper explanations.

In this case I think it would be fine to repeat the description of `create_host` and then state the difference between the two.

Reply:
thanks for explaining :) I pushed a commit (yall squash, right?) that goes through all parameters in order instead of pointing to another method